styles/readwise/catppuccin.user.css

```diff
@@ -0,0 +1,197 @@
+/* ==UserStyle==
+@name Readwise Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/readwise
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/readwise
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/readwise/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Areadwise
+@description Soothing pastel theme for Readwise
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document url-prefix('https://readwise.io/')
```

Comment:
The problem is that there is another Readwise Reader: read.readwise.io. And with just the domain selector this style was incorrectly applied also on Readwise Reader

Reply:
Can we theme Readwise Reader?

Reply:
Of course, I have a change already ready, just testing it out. But this is a separate project, so it will be a separate contribution in a separate PR and a separate file.

Reply:
I'd argue its the same domain so therefore under the same userstyle. I don't use Readwise though, thoughts @isabelroses?

Reply:
Just to put some light on the topic

Reader is a RSS reader:
<img width="1624" alt="Screenshot 2024-07-29 at 23 35 44" src="https://github.com/user-attachments/assets/2440d57f-a8cb-4586-8be0-ab8b4cf3556e">
Readwise is a special learning tool to learn the quotes from the read books:
<img width="1624" alt="Screenshot 2024-07-29 at 23 35 52" src="https://github.com/user-attachments/assets/ebb86a58-cfa7-4000-b36c-5a62406fa5fd">

Those are two separate products with compltely different codestyles and variables.

Reply:
Gotcha! This is good for now then.

Reply:
> I'd argue its the same domain so therefore under the same userstyle. I don't use Readwise though, thoughts @isabelroses?

I think the most important thing is themeing the "app" and not the landing page. This is what we have for graphite and some others alike.

Reply:
Yep, both Readwise and Readwise Reader and two separate apps